client/reader/daily-post/index.jsx

```diff
@@ -90,13 +91,21 @@ class DailyPostButton extends React.Component {
 	}
 
 	pickSiteToPostTo( siteSlug ) {
+		recordAction( 'daily_post_challenge' );
+		recordGaEvent( 'Clicked on Daily Post challenge' );
+		recordTrack( 'calypso_reader_daily_post_challenge_to_site' );
```

Comment:
maybe name this `calypso_reader_daily_post_challenge_site_picked` to keep with the tracks naming convention?